webpack.config.js

```diff
@@ -35,14 +35,6 @@ module.exports = {
         ],
         loader: 'babel'
       },
-       {
-        test: /\.glsl$/,
```

Comment:
yep, I've seen that but as we are using it for dev and I didn't see any noticeable performance hit, I thought it was not worth the effort. Also I wanted prod and dev build to stay as close as possible.

I did the measurement again, and this time my build is 6sec longer! So I guess the warning is real :-) I'm amending this right now.